### PR TITLE
PDF export: Add option 'Show only changed paragraph'

### LINKED ISSUE #3470: PDF export add option to hide text before and after change

### PR DESCRIPTION
**Pull request by m-schieder**
* hide preamble and changedTitle when it is an amendment

resolves #3470

### CODE REVIEW

**Review by bastianjoel — APPROVED** [review]
Beautiful

**Review by luisa-beerboom — APPROVED** [review]
This is great!

File: client/src/app/domain/models/motions/motions.constants.ts [hunk 1]
```diff
@@ -30,7 +30,8 @@ export const MOTION_PDF_OPTIONS = {
     Attachments: `attachments`,
     AddBreaks: `addBreaks`,
     Header: `header`,
-    ContinuousText: `continuousText`
+    ContinuousText: `continuousText`,
+    OnlyChangedParagraph: `onlyChangedParagraph`
```
Comment (emanuelschuetze) [inline client/src/app/domain/models/motions/motions.constants.ts:34]: please use better name: OnlyChangedLines

File: client/src/app/site/pages/meetings/pages/motions/components/motion-export-dialog/components/motion-export-dialog/motion-export-dialog.component.html [hunk 2]
```diff
@@ -101,6 +101,9 @@ <h1 mat-dialog-title>{{ 'Export motions' | translate }}</h1>
                     <mat-button-toggle value="continuousText" #continuousText>
                         <span>{{ 'Continuous text' | translate }}</span>
                     </mat-button-toggle>
+                    <mat-button-toggle value="onlyChangedParagraph">
+                        <span>{{ 'Only changed paragraph' | translate }}</span>
```
Comment (emanuelschuetze) [inline client/src/app/site/pages/meetings/pages/motions/components/motion-export-dialog/components/motion-export-dialog/motion-export-dialog.component.html:105]: better wording: 'Short form for amendments'

**Review by emanuelschuetze — CHANGES_REQUESTED** [review]

**Review by bastianjoel — APPROVED** [review]

**Pull request merged**